app/providers/atmosphere/optimization_strategy/default.rb

```diff
@@ -30,6 +30,18 @@ def can_scale_manually?
         false
       end
 
+      def monitor_vms?
+        false
+      end
+
+      def handle_event(hash)
```

Comment:
Unused method argument - `hash`. If it's necessary, use `_` or `_hash` as an argument name to indicate that it won't be used. You can also write as `handle_event(*)` if you want the method to accept any arguments but don't care about them.